src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -331,6 +336,8 @@ bool CryptoManagerImpl::set_certificate(const std::string& cert_data) {
     return false;
   }
 
+  // TODO(AKalinich): remove all lines below till the end of function when
```

Comment:
Can we please remove the todo, as we can't merge these comments into develop?

Reply:
@mrapitis sure, but we should not forget to remove redundant lines of code after these two pull requests merge
Fixed in https://github.com/smartdevicelink/sdl_core/pull/2218/commits/389a64f182249a4154554788f320575a697e2092